internal/providers/terraform/aws/testdata/neptune_cluster_instance_test/neptune_cluster_instance_test.golden

```diff
@@ -0,0 +1,22 @@
+
+ Name                                                  Monthly Qty  Unit                  Monthly Cost 
+                                                                                                       
+ aws_neptune_cluster.default                                                                           
+ ├─ Storage                                       Monthly cost depends on usage: $0.12 per GB          
+ └─ I/O                                           Monthly cost depends on usage: $0.23 per 1M request  
+                                                                                                       
+ aws_neptune_cluster_instance.dbR4LargeCount2[0]                                                       
+ └─ Database instance (on-demand, db.r4.large)                 730  hours                      $297.84 
+                                                                                                       
+ aws_neptune_cluster_instance.dbR4LargeCount2[1]                                                       
+ └─ Database instance (on-demand, db.r4.large)                 730  hours                      $297.84 
+                                                                                                       
+ aws_neptune_cluster_instance.dbT3Medium                                                               
+ └─ CPU credits                                              7,300  vCPU-hours                 $817.60 
+                                                                                                       
+ aws_neptune_cluster_instance.dbT3WithoutUsage                                                         
+ └─ CPU credits                                   Monthly cost depends on usage: $0.11 per vCPU-hours  
```

Comment:
In AWS Neptune's pricing page I found this: 
>  CPU Credits are charged at $0.15 per vCPU-Hour. The CPU Credit pricing is the same for T3 instance across all regions. 

What means that this cost component must be $0.15 per CPU Credit in all regions, but in our DB we have different prices for it. @alikhajeh1 / @tim775, what do you think about it? Should we be consistent with pricing page or it's ok?